mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
It is predictable indeed, but I was just thinking that it's convenient for the user to get the `event_id` since the string description already exists. Otherwise, they will have to construct the dictionary manually. But it's okay if you don't have the bandwidth to implement it or you think it's unnecessary.